yarn.lock

```diff
@@ -2876,7 +2876,7 @@ jest-pnp-resolver@^1.2.1:
 
 jest-regex-util@^24.3.0, jest-regex-util@^24.9.0:
   version "24.9.0"
-  resolved "https://registry.yarnpkg.com/jest-regex-util/-/jest-regex-util-24.9.0.tgz#c13fb3380bde22bf6575432c493ea8fe37965636"
+  resolved "https://registry.npmjs.org/jest-regex-util/-/jest-regex-util-24.9.0.tgz#c13fb3380bde22bf6575432c493ea8fe37965636"
```

Comment:
Wat?

Reply:
Seems like I've set the registry to npm after having some problems when installing things long time ago. I myself have even forgotten about this and noone has commented on this during my PRs 🤷‍♂ I recall yarn even recommending to set this in some issues as that yarn registry is just a proxy to npm or something. Dunno though - gonna just change my global config back to default and fix this here.